src/components/Grid.jsx

```diff
@@ -33,6 +33,22 @@ class Grid extends React.Component {
     }
   }
 
+  getLines () {
+    let lines = []
+
+    for (let i = 0; i < this.props.hours; i++) {
```

Comment:
The first thing, extract the logic inside for loop into a separate function that will return single line. Then you can use e.g. [R.times](http://ramdajs.com/0.18.0/docs/#times) like this:

```js
const getLines = () => R.times(createLine, this.props.hours)
```